include/fmt/format.h

```diff
@@ -391,11 +391,14 @@ class uint128_fallback {
       hi_ += (lo_ < n ? 1 : 0);
       return *this;
     }
-#if FMT_HAS_BUILTIN(__builtin_addcll)
+#if FMT_XLC_VERSION && defined(_ARCH_PWR9)
+    lo_ = __addex(lo_, n, 0, &carry);
+    hi_ += carry;
```

Comment:
Do we need this optimization? I suggest not introducing it unless not having it is causing some real perf problems. The fallback normally generates identical code: https://godbolt.org/z/xMGE5aE4T.

Reply:
__addex works only for IBM Power9 platforms. Anyway, it is not critical